docs/plotting.md

```diff
@@ -194,7 +195,13 @@ Sample configuration with inline comments explaining the process:
             "RSI": {
                 'rsi': {'color': 'red'},
             }
-        }
+        },
+	'fill_area': {
+	    "Ichimoku Cloud": { 
+		'traces': ('senkou_a', 'senkou_b'),
+		'color': 'rgba(0,176,246,0.2)', 
+	     },
```

Comment:
Are `senkou_a` and `senkou_b` necessary in main_plot as well?

or are they implicitly added to the main plot?
Neither is a problem, but it should be documented...